src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5890,6 +5890,7 @@ private enum ParseTypeMode
             AfterIs,
             DefinitePattern,
             AfterOut,
+            AfterRef,
```

Comment:
I don't understand why we're seeing this change in the diff. I would expect this diff (at least in github UI) to only show the target-typed-new changes, but nothing from other merges.
Am I missing something?

Reply:
This is added in this PR, to avoid cascading diagnostics e.g. for `new ref`